### ISSUE TITLE
Error on save undefined method named "addLogger"

### ISSUE THREAD

**Issue by Petrucheqa** [DATA-DOG/DataDogAuditBundle#84]
Hello, 

I'm facing this error on entity edit : 
![Capture d’écran 2021-12-29 à 11 19 21](https://user-images.githubusercontent.com/17081646/147653613-f98c6c77-46a3-48f4-b8b1-24a58afdc3b3.png)

Here is SF configuration : 
![Capture d’écran 2021-12-29 à 11 18 59](https://user-images.githubusercontent.com/17081646/147653658-1383811d-1012-4513-b761-248f37e40fb5.png)

bundle config :
```
data_dog_audit:
    audited_entities:
        - App\Entity\Formation
```

Tell me if you want more informations about the "bug".

**Comment by natewiebe13** [comment]
Previously there wasn't a version constraint on `doctrine/dbal` which is likely the cause of this. Can you update to v0.1.21 and let us know if you still have this issue?

**Comment by digitalshow** [comment]
Seems to be removed in `doctrine/dbal`
https://github.com/doctrine/dbal/blob/3.2.x/UPGRADE.md#deprecated-sqllogger-and-its-implementations
https://github.com/doctrine/dbal/commit/06dc97e0faf2a2012e2c8e0c07289a99a806301f#diff-ede4922d6cee6c8394a3d85443d4a4e4b0faf43fd34190fb1d82193116f4d928

**Comment by natewiebe13** [comment]
Correct, this was added in https://github.com/DATA-DOG/DataDogAuditBundle/commit/519d1574c44e5b889e509bd66c2200f400fb5dce.

My guess here is the latest version of this bundle is not being used.
